### DISCUSSION TITLE
UEFI

### DISCUSSION THREAD

**Discussion by Kosmotello** [mikopbx/Core#276] [Q&A] [answered]
Столкнулся с проблемой запуска инсталяции из под UEFI. Насколько я понял ядро собрано без поддержки UEFI - скажите, планируется ли в дальнейшем поддержка данной технологии?

**Comment by boffart** [comment, marked as answer]
Уже были попытки реализовать поддержку UEFI. Пока положительного результата нет, функционал требует больших трудозатрат. 
В будущем, постараемся реализовать. 
На текущий момент если UEFI необходим используйте Debian в связке с [docker](https://wiki.mikopbx.ru/setup#docker).
